austrakka/components/project/dataset/__init__.py

```diff
@@ -73,7 +72,7 @@ def get_active_dataset_entry_list(abbrev: str, out_format: str):
 @click.option('-id',
               '--dataset-id',
               help='dataset to disable', )
-@opt_abbrev(help="Project Abbreviation")
+@click.argument('abbrev', type=str)
```

Comment:
My instinct is to make this a parameter (--project) since the primary entity we're operating on is the dataset. Given --project is long-form, maybe leave it for now and we can refactor everything at once